packages/create-fuels/package.json

```diff
@@ -22,6 +22,7 @@
     "@iarna/toml": "^2.2.5",
     "chalk": "4",
     "commander": "^9.4.1",
+    "execa": "^8.0.1",
```

Comment:
What led you here?

Reply:
Wasn't a more "native" `exec<Sync>` or `spawn<Sync>` enough for the job?

Reply:
Yep, reverted this. I added it for other reasons which are not valid anymore